<launch>
  <arg name="rsbb_host" default="10.255.255.255"/>
  <arg name="rsbb_port" default="6666"/>
  <arg name="benchmarks_file" default="$(find roah_rsbb)/config/benchmarks.yaml"/>
  <arg name="schedule_file" default="$(find roah_rsbb)/config/schedule.yaml"/>
  <arg name="passwords_file" default="$(find roah_rsbb)/config/passwords.yaml"/>
  <arg name="log_dir" default="$(find roah_rsbb)/log"/>

  <include file="$(find roah_rsbb)/launch/roah_rsbb_public.launch"/>

  <node pkg="roah_rsbb" type="core" name="roah_rsbb_core" respawn="true">
    <param name="rsbb_host" type="string" value="$(arg rsbb_host)"/>
    <param name="rsbb_port" type="int" value="$(arg rsbb_port)"/>
    <param name="benchmarks_file" type="string" value="$(arg benchmarks_file)"/>
    <param name="schedule_file" type="string" value="$(arg schedule_file)"/>
    <param name="passwords_file" type="string" value="$(arg passwords_file)"/>
    <param name="log_dir" type="string" value="$(arg log_dir)"/>
  </node>
  
  <include file="$(find roah_devices)/launch/run_dummy.launch"/>

</launch>
